### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by saschatimme**
The repo looked quite sad without a proper README. I think this is now better than nothing. I also wanted to link Michaels new notebook but couldn't find it on the OSCAR homepage. Do you have a link for me @sebasguts ?

### CODE REVIEW

**Comment by saschatimme** [comment]
But the polymake example at https://oscar.computeralgebra.de/example/ is still the old notebook

**Comment by sebasguts** [comment]
Yes, because I did not get to update it yet. Will probably happen next week.

**Comment by saschatimme** [comment]
Well then I just add this later. Is the rest fine?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,83 @@
...
 
-[![Build Status](https://travis-ci.org/sebasguts/Polymake.jl.svg?branch=master)](https://travis-ci.org/sebasguts/Polymake.jl)
+Julia package for using [polymake](https://polymake.org/doku.php), a software for research in polyhedral geometry.
+This package is developed as part of the [OSCAR](https://oscar.computeralgebra.de) project.
 
-[![Coverage Status](https://coveralls.io/repos/sebasguts/Polymake.jl/badge.svg?branch=master&service=github)](https://coveralls.io/github/sebasguts/Polymake.jl?branch=master)
+## Installation
 
-[![codecov.io](http://codecov.io/github/sebasguts/Polymake.jl/coverage.svg?branch=master)](http://codecov.io/github/sebasguts/Polymake.jl?branch=master)
+The current version relies on an unreleased version of polymake. A compatible version is available at [polymake/polymake#snapshots](https://github.com/polymake/polymake/tree/Snapshots) and it can be compiled as follows where `GIT_FOLDER` and `INSTALL_FOLDER` have to be substituted with your favorite folder.
```
Comment (sebasguts) [inline README.md]: Please add a link to the Polymake website showing all the dependencies one needs to compile polymake

File: README.md [hunk 2]
```diff
@@ -1,7 +1,83 @@
...
+git clone git@github.com:polymake/polymake $POLYMAKE_GIT
+cd $POLYMAKE_GIT
+git checkout Snapshots
+./configure --prefix=$POLYMAKE_INSTALL
+ninja -C build/Opt
+ninja -C build/Opt install
+export POLYMAKE_CONFIG=$POLYMAKE_INSTALL/bin/polymake-config
+```
+
+Note that the compilation can take.
```
Comment (sebasguts) [inline README.md]:
```suggestion
Note that polymake might take some time to compile.
```

File: README.md (hunk 1 above)
Comment (saschatimme) [reply README.md]: good point added it.

**Pull request merged**